### PR TITLE
Updated theanarchistlibrary

### PR DESCRIPTION
**Pull request by Devices**
Added two new subdomains. Also added three other domains that provide the library in different languages, but are still part of the same project.

### CODE REVIEW

File: src/chrome/content/rules/TheAnarchistLibrary.org.xml [hunk 1]
```diff
@@ -13,7 +13,12 @@
 	<target host="ru.theanarchistlibrary.org" />
 	<target host="es.theanarchistlibrary.org" />
 	<target host="sv.theanarchistlibrary.org" />
+	<target host="da.theanarchistlibrary.org" />
+	<target host="fr.theanarchistlibrary.org" />
 	<target host="www.theanarchistlibrary.org" />
+	<target host="www.anarhisticka-biblioteka.org" />
+	<target host="anarchistischebibliothek.org" />
+	<target host="anarhisticka-biblioteka.net" />
```
Comment (J0WI) [inline src/chrome/content/rules/TheAnarchistLibrary.org.xml]: Please always add the `www` and non-www aliases of those three new hosts.

Comment (Devices) [inline src/chrome/content/rules/TheAnarchistLibrary.org.xml]: Even if they redirect to the non-www? www.anarchistischebibliothek.org redirects to anarchistischebibliothek.org and anarhisticka-biblioteka.org redirects to www.anarhisticka-biblioteka.org.

Comment (J0WI) [inline src/chrome/content/rules/TheAnarchistLibrary.org.xml]: We want to secure everything, even redirects ;)

Comment (Devices) [inline src/chrome/content/rules/TheAnarchistLibrary.org.xml]: Makes sense. I hope it's good like this.

Comment (J0WI) [inline src/chrome/content/rules/TheAnarchistLibrary.org.xml]: LGTM, Thanks!

**Pull request merged**